src/pages/_app.tsx

```diff
@@ -99,23 +100,25 @@ const App = ({
           }}
         >
           <SWRConfig value={{ fetcher: fetcherForSWR }}>
-            <WagmiConfig config={config}>
-              <PostHogProvider>
-                <IntercomProvider>
-                  <ExplorerProvider>
-                    <AppErrorBoundary>
-                      <Component {...pageProps} />
-                    </AppErrorBoundary>
+            <XMTPProvider>
```

Comment:
Let's remove this provider and just use it inside the `MessagesSection` component & `SendNewMessage` modal. That way we won't load it by default, only when we actually need it.